### PR TITLE
Fix backend getcoordinates in location module

### PR DESCRIPTION
**Pull request by jeroendesloovere**

### CODE REVIEW

**Comment by jeroendesloovere** [comment]
Fixes #1672

File: src/Backend/Modules/Location/Engine/Model.php [hunk 1]
```diff
@@ -138,11 +138,8 @@ public static function getCoordinates(
         // define address
         $address = implode(' ', $item);
 
-        // define Google Maps API key
-        $apikey = BackendModel::get('fork.settings')->get('Core', 'google_maps_key');
-
         // define url
-        $url = 'http://maps.googleapis.com/maps/api/geocode/json?address=' . rawurlencode($address) . '&key=' . $apikey;
+        $url = 'https://maps.googleapis.com/maps/api/geocode/json?address=' . rawurlencode($address);
```
Comment (carakas) [inline src/Backend/Modules/Location/Engine/Model.php]: why did you remove the key?

**Comment by carakas** [comment]
your builds are failing, since this is such a minor change it might be better to close this one and create a new pull request that doesn't have 14 merges in it :)

File: src/Backend/Modules/Location/Engine/Model.php (hunk 1 above)
Comment (carakas) [inline src/Backend/Modules/Location/Engine/Model.php]: if it is the wrong key, shouldn't we add the correct key

Comment (jeroendesloovere) [inline src/Backend/Modules/Location/Engine/Model.php]: We need two keys if we want to do it right:
- **browser key** to show the google maps correctly
- **server key** to get the geocoordinates (in BackendLocationModel), but this one is optional. That's why I removed it in BackendLocationModel.

Currently we only use the browser key to show the "google maps".

**Comment by jeroendesloovere** [comment]
schermafbeelding 2016-08-18 om 15 35 54

**Comment by carakas** [comment]
@jeroendesloovere 
screen shot 2016-08-21 at 17 05 42
https://travis-ci.org/forkcms/forkcms/builds/152654762

**Comment by bartdc** [comment]
A Server key can also show a map, besides e.g. get coordinates.

So if we use a Server key in the Backend and the same key in the Frontend, a Browser key is unnecessary.
